images/examples/job-worker/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2023 Steffen Vogel, OPAL-RT Germany GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+[project]
+name = "seguro-example-job-worker"
+version = "0.1.0"
+dependencies = [
+    "seguro-platform@git+https://github.com/SEGurO-Projekt/Platform"
+]
+
+requires-python = ">= 3.11"
+
+[project.scripts]
+worker = "job_worker.main:main"
```

Comment:
Fixed in https://github.com/SEGuRo-Projekt/Platform/pull/76/commits/9afcc662be7d3d1d9b13564af9645edde82202e3

But I used a hyphen instead of an underscore (`job-worker`) as this more harmonized with the other commands we already have.